cosmo/graphqlclient.py

```diff
@@ -1,10 +1,44 @@
 import json
 from string import Template
+from urllib.parse import urlencode
 
 import requests
 
 
 class GraphqlClient:
```

Comment:
Our client does more than GraphQL now, since we also have REST stuff going on for version check.

__Maybe we should rename it?__ Either NetboxClient or NetboxContext + Netbox3Strategy & Netbox4Strategy (we're kinda using the strategy pattern - I know it bc I solved the same problem in ntd ztp crawler :stuck_out_tongue_winking_eye:)

Reply:
Done!